flamedisx/likelihood.py

```diff
@@ -7,6 +7,11 @@
 import tensorflow as tf
 import typing as ty
 
+#### PL
+import pdb as pdb
+import scipy.optimize as spo
+from scipy.optimize import NonlinearConstraint
+#### end PL
```

Comment:
Unused, I guess `likelihood.py` can be left as it was. At least until we remove the tilt based interval classes at which point we can clean up the interface in `Likelihood`